doc/content/with_conda.md

```diff
@@ -21,16 +21,19 @@ OpenMC separately, or using its conda environment (i.e., do
 
 !alert! note title=tldr
 
-All that you need to compile Cardinal is:
+Once your `cardinal` conda environment is set up, all you need to do to compile Cardinal is:
 
 ```
-conda activate moose
+conda activate cardinal
+conda install -c conda-forge openmc # Install OpenMC from conda-forge
```

Comment:
Can you please update this documentation so that it does not rely on the OpenMC conda environment? Let's stick with the original MOOSE conda environment since it includes libMesh & wasp & petsc and simultaneously has all the OpenMC dependencies already